src/wp-admin/includes/class-wp-plugins-list-table.php

```diff
@@ -1149,6 +1161,17 @@ public function single_row( $item ) {
 
 					echo '</div>';
 
+					/**
+					 * Add text at end of plugin row meta.
+					 *
+					 * @since 6.4.0
+					 *
+					 * @param string null         Text to add after plugin row meta.
+					 * @param string $plugin_file Refer to 'plugin_row_meta' filter.
+					 * @param array  $plguin_data Refer to 'plugin_row_meta' filter.
+					 */
+					echo apply_filters( 'post_plugin_row_meta', null, $plugin_file, $plugin_data );
```

Comment:
```suggestion
					/**
					 * Filters text to add after plugin row meta.
					 *
					 * @since 6.4.0
					 *
					 * @param string $after       Text to add after plugin row meta. Default empty string.
					 * @param string $plugin_file Refer to 'plugin_row_meta' filter.
					 * @param array  $plugin_data Refer to 'plugin_row_meta' filter.
					 */
					$after_row_meta = apply_filters( 'after_plugin_row_meta', '', $plugin_file, $plugin_data );

					if ( '' !== $after_row_meta ) {
						echo esc_html( $after_row_meta );
					}
```
Suggested changes:
- Change the summary to start with "Filters..." and to align language with the parameter description.
- Only use `string` for the type, with an empty string being the default value.
- Add missing parameter name and correct a typo in the docblock.
- Rename `post_` to `after_` to avoid confusion with post-related hooks.
- Check for an empty string. Otherwise, escape and output.

Reply:
Actually, this may be better as an Action hook instead, with the escaping happening in the callback.

Reply:
That would work

Reply:
```suggestion
					/**
					 * Fires after plugin row meta.
					 *
					 * @since 6.4.0
					 *
					 * @param string $plugin_file Refer to {@see 'plugin_row_meta'} filter.
					 * @param array  $plguin_data Refer to {@see 'plugin_row_meta'} filter.
					 */
					do_action( 'after_plugin_row_meta', $plugin_file, $plugin_data );
```